Source/iop/Iop_McServ.h

```diff
@@ -198,6 +198,11 @@ namespace Iop
 		static const char* m_mcPathPreference[2];
 		std::string m_currentDirectory;
 		CPathFinder m_pathFinder;
+
+		// Keeps track, if the memory card in
+		// a given slot has already been read,
+		// or if it is a newly inserted card.
+		std::vector<bool> m_knownMemoryCards;

```

Comment:
This doesn't need to be dynamic. All the code assumes there's two ports. It would probably be a good occasion to add a `MAX_PORTS` enum inside the class and use it instead of the hardcoded `2`.

Reply:
Force pushed the change, and added the enum in a separate commit